app/packages/core/src/components/Filters/CategoricalFilter.tsx

```diff
@@ -102,18 +102,25 @@ const Wrapper = ({
   const selectedSet = new Set(selected);
   const setExcluded = excludeAtom ? useSetRecoilState(excludeAtom) : null;
   const sorting = useRecoilValue(fos.sortFilterResults(modal));
-  const counts = Object.fromEntries(results);
+  const filteredResults = schema?.ftype.includes("ListField")
+    ? results.filter(([v]) => v !== null)
+    : results;
+  const counts = Object.fromEntries(filteredResults);
   let allValues: V[] = selected.map<V>((value) => ({
     value,
     count: counts[String(value)] ?? 0,
   }));
+  allValues = schema?.ftype.includes("ListField")
```

Comment:
I don't think this is necessary as these extra values represent items that have already been selected